### PR TITLE
fix for new milight hub 1.11+

### PR DESCRIPTION
**Pull request by normen**
This still to be tested patch adds support for getting the list of lights from milight-hub v 1.11+

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -121,12 +121,29 @@ class MiLightHubPlatform {
...
+        } else {
+          const aliases_path = '/aliases';
+          platform.apiCall(aliases_path).then(response => {
+            if (response) {
+              const aliases = JSON.parse(response);
+              // TODO: check if we have to paginate (no API docs for that?)
+              platform.log(aliases);
+              for (var alias in aliases.aliases) {
+                //var lightInfo = { name: alias.alias, device_id: alias.device_id, group_id: alias.group_id, remote_type: alias.device_type, uid: '0x' + alias.device_id.toString(16).toUpperCase() + '/' + alias.device_type + '/' + alias.group_id };
+                var lightInfo = { name: alias.alias, device_id: alias.device_id, group_id: alias.group_id, remote_type: alias.device_type, uid: '0x' + alias.device_id + '/' + alias.device_type + '/' + alias.group_id };
```
Comment (mlask) [inline index.js]: This will, eventually, throw TypeError later, because `alias` contains only index from table, not object.

Comment (normen) [reply index.js]: I don't have 1.11 installed locally as I don't have a free esp to test right now, the log output would help more. `aliases` has different content in different situations is what you're saying?

Comment (mlask) [reply index.js]: I'm on 1.11.2 and `.../aliases` returns
```
{"aliases":[{"alias":"devicename","id":1,"device_id":123,"group_id":4,"device_type":"rgb_cct"}],"page":1,"count":1,"num_pages":1}
```

I've had to add simple fix after line 138 (`alias = aliases.aliases[alias];`) to make it working again.

Comment (normen) [reply index.js]: Ah, the iterator, darn, I'm jumping between languages too much ^^ Thanks, that should be an easy fix then. Now I just need to check if the whole pagination is real.

**Pull request merged**